Source/WebCore/page/LocalFrameView.cpp

```diff
@@ -3919,6 +3955,9 @@ void LocalFrameView::scheduleResizeEventIfNeeded()
         return;
     }
 
+    invalidateScrollAnchoringElement();
+    updateScrollAnchoringElement();
```

Comment:
Seems wrong to do this in a function which is about scheduling the resize event. Move this to another resize bottleneck.

Reply:
This will need to be moved to an area that applies for all ScrollableAreas. This requires more investigation.